src/app/lib/tasks.ts

```diff
@@ -0,0 +1,6 @@
+export const unscheduledTasks = [];
```

Comment:
Regarding this file, I kept going back and forth on whether I should create a useContext for this data, or create a fake api, or create classes. This is a short term solution to storing the data temporarily so that it can be accessed in different pages.

When it comes to creating the backend, the way I envision possibly is unscheduled tasks and availability get sent to the backend, schedule gets generated in the backend, and then gets displayed in the fronted.

Once we better understand our backend design, this can be updated!